### DISCUSSION TITLE
将xml格式转为node格式的API

### DISCUSSION THREAD

**Discussion by FandyTuTu** [didi/LogicFlow#1481] [Q&A]
导出数据为xml格式，如何将xml格式转为node格式，并将其回显呢，好像没看到这样的API

**Comment by boyongjiong** [comment]
你用的什么方法转换成 xml 的？adapterIn  和 adapterOut 分别是将数据从外部格式 与 LogicFlow Data 相互转换的方法
https://beta.logic-flow.cn/tutorial/extension-adapter 可以看下这个文档

**Reply by FandyTuTu** [reply]
你说的这两个api我看到了并使用了，我的意思是，如何将保存好的xml（例：<bpmn:definitions  id="Definitions_bc8c610">.....
</bpmn:definitions>）数据回显成“流程图”。
后来我将绘制好的流程图保存为“node”节点格式数据，再将“node”节点格式数据 传给 render()方法，页面并没有显示数据对应的流程图，不知道为啥
